recipe/build.sh

```diff
@@ -26,30 +34,60 @@ cmake \
   -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES:BOOL=OFF \
   -D Trilinos_ENABLE_TESTS:BOOL=OFF \
   -D Trilinos_ENABLE_EXAMPLES:BOOL=OFF \
+  -D Trilinos_ENABLE_Teuchos:BOOL=ON \
+  -D Trilinos_ENABLE_RTOp:BOOL=ON \
+  -D Trilinos_ENABLE_Sacado:BOOL=ON \
+  -D Trilinos_ENABLE_MiniTensor:BOOL=ON \
   -D Trilinos_ENABLE_Epetra:BOOL=ON \
+  -D Trilinos_ENABLE_Zoltan:BOOL=ON \
+  -D Trilinos_ENABLE_Shards:BOOL=ON \
+  -D Trilinos_ENABLE_GlobiPack:BOOL=ON \
   -D Trilinos_ENABLE_Triutils:BOOL=ON \
   -D Trilinos_ENABLE_Tpetra:BOOL=ON \
   -D Trilinos_ENABLE_EpetraExt:BOOL=ON \
   -D Trilinos_ENABLE_Domi:BOOL=ON \
-  -D Trilinos_ENABLE_Isorropia:BOOL=OFF \
-  -D Trilinos_ENABLE_Pliris:BOOL=OFF \
+  -D Trilinos_ENABLE_Thyra:BOOL=ON \
+  -D Trilinos_ENABLE_Xpetra:BOOL=ON \
+  -D Trilinos_ENABLE_OptiPack:BOOL=ON \
+  -D Trilinos_ENABLE_Isorropia:BOOL=ON \
+  -D Trilinos_ENABLE_Pliris:BOOL=ON \
   -D Trilinos_ENABLE_AztecOO:BOOL=ON \
   -D Trilinos_ENABLE_Galeri:BOOL=ON \
   -D Trilinos_ENABLE_Amesos:BOOL=ON \
+  -D Trilinos_ENABLE_Pamgen:BOOL=ON \
+  -D Trilinos_ENABLE_Zoltan2:BOOL=ON \
   -D Trilinos_ENABLE_Ifpack:BOOL=ON \
-  -D Trilinos_ENABLE_Komplex:BOOL=ON \
   -D Trilinos_ENABLE_ML:BOOL=ON \
+  -D Trilinos_ENABLE_Belos:BOOL=ON \
+  -D Trilinos_ENABLE_ShyLU:BOOL=ON \
+  -D Trilinos_ENABLE_Amesos2:BOOL=ON \
+  -D Trilinos_ENABLE_SEACAS:BOOL=OFF \
+  -D Trilinos_ENABLE_Komplex:BOOL=ON \
   -D Trilinos_ENABLE_Anasazi:BOOL=ON \
-  -D Trilinos_ENABLE_NOX:BOOL=OFF \
-  -D NOX_ENABLE_LOCA:BOOL=OFF \
+  -D Trilinos_ENABLE_Ifpack2:BOOL=ON \
+  -D Ifpack2_ENABLE_TESTS:BOOL=OFF \
+  -D Trilinos_ENABLE_Stratimikos:BOOL=ON \
+  -D Trilinos_ENABLE_FEI:BOOL=ON \
+  -D Trilinos_ENABLE_Teko:BOOL=OFF \
+  -D Trilinos_ENABLE_Intrepid:BOOL=ON \
+  -D Trilinos_ENABLE_Intrepid2:BOOL=ON \
+  -D Trilinos_ENABLE_STK:BOOL=OFF \
+  -D Trilinos_ENABLE_Phalanx:BOOL=ON \
+  -D Trilinos_ENABLE_NOX:BOOL=ON \
+  -D NOX_ENABLE_LOCA:BOOL=ON \
+  -D Trilinos_ENABLE_MueLu:BOOL=ON \
+  -D Trilinos_ENABLE_Rythmos:BOOL=ON \
+  -D Trilinos_ENABLE_Tempus:BOOL=ON \
+  -D Trilinos_ENABLE_Stokhos:BOOL=ON \
+  -D Trilinos_ENABLE_ROL:BOOL=ON \
+  -D Trilinos_ENABLE_Piro:BOOL=ON \
+  -D Trilinos_ENABLE_Panzer:BOOL=ON \
+  -D Trilinos_ENABLE_TrilinosCouplings:BOOL=ON \
+  -D Trilinos_ENABLE_Pike:BOOL=ON \
   -D Trilinos_ENABLE_PyTrilinos:BOOL=ON \
-  -D PyTrilinos_ENABLE_TESTS:BOOL=ON \
-  -D PyTrilinos_ENABLE_EXAMPLES:BOOL=ON \
-  -D PyTrilinos_INSTALL_PREFIX:PATH=$PREFIX \
   $SRC_DIR
 
 make -j $CPU_COUNT
-
+$PYTHON packages/PyTrilinos/util/configFix.py
```

Comment:
This is unnecessary because the files that this script is patching was already packaged in `trilinos` and therefore not changed by the presence of pytrilinos

Reply:
The content of the files that this script is patching *does* change, and the change is a bug that needs to be addressed.

Reply:
So you are saying that the file will not get overwritten on install, even though the content of the file changes?

Reply:
Yes.